docs/source/docfiles/markdown/QUICKSTART.md

```diff
@@ -60,6 +63,20 @@ bash ./resources/vm/setup_farmvibes_ai_vm.sh
 
 You might needed to restart your shell session once the script finishes.
 
+## Optional: Restore files with Git LFS
```

Comment:
Is this really optional?

Reply:
Yes, you are right. I'll remove it. I was thinking it was optional in case the user doesn't have Git LFS installed when cloning the repo.